plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/ParquetTester.java

```diff
@@ -816,7 +817,7 @@ private static void writeParquetColumnTrino(
                         .build(),
                 compressionCodec,
                 "test-version",
-                Optional.of(DateTimeZone.getDefault()),
+                Optional.of(UTC),
```

Comment:
are tests missing here for non-utc?

Reply:
Now that value of timezone takes effect test would fail as it tries to adjust the time with the given timezone.